mods/voidmulti.sc2mod/base.sc2data/gamedata/effectdata.xml

```diff
@@ -724,14 +736,15 @@
         <PeriodicOffsetArray value="0,-11,0"/>
         <PeriodicOffsetArray value="0,-12,0"/>
         <PeriodicEffectArray index="0" value="LurkerMPCU"/>
+        <PeriodicValidator value=""/>
     </CEffectCreatePersistent>
     <CEffectDamage id="LurkerMPDamage">
         <Kind value="Ranged"/>
         <AttributeBonus index="Armored" value="10"/>
     </CEffectDamage>
     <CEffectEnumArea id="LurkerMPSearch">
         <AreaArray index="0" Radius="0.5"/>
-        <ValidatorArray value="WeaponInRange"/>
+        <ValidatorArray index="0" value="WeaponInRange"/>
```

Comment:
Lurker attacks will no longer be blocked by certain low-ground terrain features

This change removes the CliffLevelGE1 validator.
For some reason this validated that the target unit's point was not on the cliff level down the deep cliffs.
It should have prevented units from receiving damage when the spine's center point is down the deep cliff levels and a unit is close to it.
Now a unit close to the cliff still receives damage when the center of the area damage is down in the deep cliff.
Is this assessment correct?
Are there other terrain features like doodads that alters the cliff level to 0 causing this issue? It would be great to have an exact example scenario